.gitignore

```diff
@@ -1 +1,29 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+ExampleApp/Assets/Plugins/Android/OptimoveUnityPlugin-debug.aar
```

Comment:
this gitignore has few things:
1. there is a separate gitignore for ExampleApp
2. what’s the intention for aar? It’s in gitignore, but present in the PR. Perhaps we can include it, then running the ExampleApp is slightly simpler. So, to run ExampleApp would only need to set credentials -----> Remove from gitignore? Also, the aar we add mist be release, not debug
3. what is google-services-desktop.sjon in StreamingAssets?
4. why this needed  unity-sdk/AndroidPlugin/OptimoveUnityPlugin/google-services.json ? are we able to receive push in AndroidPlugin project? (if yes, cool)
5. .gitignore — gitignore adding self to .gitignore? o_o
6. ExampleApp/firstTest_BurstDebugInformation_DoNotShip/tempburstlibs/armeabi-v7a/lib_burst_generated.txt this would ignore only certain single architecture, could it generate files with other names? But now i removed burst compilation, can get rid of it